db/seeds/production.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+AccessionNumber.delete_all
```

Comment:
Would this delete all when the app is deployed?

Reply:
Only the first time, which we will do with a cap task. The seeds do not run unless you do `rails db:seed`